_data/blocks/big-data-and-data-engineering.pt_BR.yaml

```
@@ -67,3 +69,28 @@ alura-contents:
   - type: COURSE
     title: "Curso Spark: streaming de dados"
     link: https://cursos.alura.com.br/course/spark-streaming-dados
+  - type: COURSE
+    title: "Curso Spark: streaming de dados"
+    link: https://cursos.alura.com.br/course/spark-streaming-dados
+  - type: COURSE
+    title: "Formação inicial de engenharia de dado"
+    link: https://cursos.alura.com.br/formacao-iniciando-engenharia-dados
+movies:
+  - type: FILME
```

Todos os `type: FILME`, troca pra `MOVIE`
```suggestion
  - type: MOVIE
```